.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+# Pull Request template  
+
+## Description  
+Fixes # (Issue-number)  
+
+A brief summary of the contribution and the problem it solves. Try to keep the description short and only include relevant details.
+
+## Type of contribution  
```

Comment:
It's not required, usually the type of contribution is mentioned in the Title section🙂